Bottom: 55b2a8b4064bed2c301d5beb74f44221a926cd67
Top:    ceb35731b4a2b09edff6b36b831ea8cd679cf821
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-08 09:20:48 -0500

Refresh of tmp-wiki-database

---
diff --git a/roles/docker-mediawiki/defaults/main.yml b/roles/docker-mediawiki/defaults/main.yml
index 94ef795b..8a608ebd 100644
--- a/roles/docker-mediawiki/defaults/main.yml
+++ b/roles/docker-mediawiki/defaults/main.yml
@@ -18,7 +18,7 @@ wiki_image_repo: https://github.com/novatechweb/docker-mediawiki.git
 
 # the name of the image being duilt and used for the container
 wiki_image_name: '{{ docker_registry_username }}/wiki'
-wiki_db_image: 'mysql:5'
+wiki_db_image_name: 'mysql:5'
 
 # the name of the container being started
 wiki_container_name: '{{ docker_name_prefix }}wiki'
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index 67b60ab5..81694f34 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -102,7 +102,7 @@
     name: '{{ wiki_db_container_name }}'
     detach: true
     restart_policy: always
-    image: '{{ wiki_db_image }}'
+    image: '{{ wiki_db_image_name }}'
     volumes:
       - '{{ wiki_db_dv_name }}:/var/lib/mysql:z'
     env:
